### PR TITLE
Project 1 added

### PR DESCRIPTION
**Pull request by ishukhanchi**
Kindly check.

### CODE REVIEW

**Comment by gkanishk** [comment]
Hey @ishukhanchi ,
- Can you elaborate the description part a bit also,
- Try to change the title of your project it initially shows "React App" so please update accordingly.
- and if you could do some styling will look better.
Best,
Kanishk

File: content/participants/ttjc@ishukhanchi/index.md [hunk 1]
```diff
@@ -35,7 +35,18 @@ It would be good if someone tells me how to be more productive and what all one
 
 ### projects
 
-Not a lot of projects are done. I will keep updating it with real links when done.
+My projects:
+
+#### ShopList
+
+_description_ An App for making shopping lists.
```
Comment (gkanishk) [inline content/participants/ttjc@ishukhanchi/index.md:42]: please elaborate this part

**Comment by gkanishk** [comment]
@ishukhanchi looks good now!!
One thing I noticed when we add items to the list each second-time item is not reflected that is only reflected in an odd number of times, so can you have a look!!

**Comment by ishukhanchi** [comment]
> @ishukhanchi looks good now!!
> One thing I noticed when we add items to the list each second-time item is not reflected that is only reflected in an odd number of times, so can you have a look, also if you could add any background that will look good.

Yeah. I saw that and tried but still not getting why that is happening.

**Comment by gkanishk** [comment]
Hey @ishukhanchi ,
Try to add toggleshow inside the add function so that when add function is called automatically data is shown
image
Try this and let me know if this works else you can DM me on telegram or discord my username remains same gkanishk.

**Pull request merged**